src/backend/distributed/transaction/transaction_management.c

```diff
@@ -156,6 +140,25 @@ InitializeTransactionManagement(void)
 }
 
 
+/*
+ * BeginCoordinatedTransaction begins a coordinated transaction. No
+ * pre-existing coordinated transaction may be in progress./
+ */
+static void
+BeginCoordinatedTransaction(void)
```

Comment:
Maybe move this back to not get `git blame`d

Reply:
Done.

Reply:
Actually, I'm going to leave this (though I'll keep it in a separate commit. During coding I saw this function was only called from within this file, so I moved it down with the other static functions and marked it as such.

By keeping it in a separate commit with a good message, I doubt we'll have any mistaken attribution (and more advanced git users would probably be using flags to ignore moves anyhow).